ctags

```diff
@@ -0,0 +1,8 @@
+--regex-ruby=/(^|[:;])[ \t]*([A-Z][[:alnum:]_]+) *=/\2/c,class,constant/
+--regex-ruby=/^[ \t]*attr_(reader|writer|accessor) (:[a-z0-9_]+, )*:([a-z0-9_]+)/\3/A,attr/
+--regex-ruby=/^[ \t]*context ['"](.*)['"].* do/\1/C,rspec context/
```

Comment:
Sorry for being late to respond @gabebw. 

1. Yes, that's exactly right.
2. You can navigate to them with `:tag #create` to jump to the `#create` describe-block. And also `:tag with invalid payload` to jump to those matching contexts. It's mostly useful for me to do that with either [pick] or [ctrlp]

[pick]: https://github.com/thoughtbot/pick.vim#picktag
[ctrlp]: https://github.com/ctrlpvim/ctrlp.vim